templates.json

```diff
@@ -312,8 +312,8 @@
     },
     {
       "id": "verify-sna",
-      "name": "verify-sna",
-      "description": "Sample Backend Silent Network Auth"
+      "name": "Verify Silent Network Auth Sample Backend",
+      "description": "Starts and checks SNA verifications and shows their status in the dashboard as they come in"
```

Comment:
what about:

`
Backend to communicate with Verify API. This backend starts and validates login request against the Verify API by using the SNA channel functionality. The results will be presented on the UI dashboard

`

Just a suggestion, maybe the current description is too technical for someone that doesn't have the context.

Reply:
Yes sounds a lot better, thanks